lib/json_info_codec.dart

```diff
@@ -72,8 +72,20 @@ class JsonToAllInfoConverter extends Converter<Map<String, dynamic>, AllInfo> {
     result.program = parseProgram(json['program']);
 
     if (json['deferredFiles'] != null) {
-      result.deferredFiles =
+      final deferredFilesMap =
```

Comment:
(optional) rather than updating and casting the map in-place, I wonder if this merits creating a new map instead. We could possibly add a `parseDeferredMap` if it helps make it more structured.

Reply:
Talked offline, we can go with this approach right now. In the future we should have better typing for this field, which will make this a non-issue